src/styles/_common.scss

```diff
@@ -171,6 +173,27 @@
   padding-right: 15px;
 }
 
+.link-remove-style {
+  a {
+    color: $title-color;
+    text-decoration: none;
+  }
+
+  a:hover {
+    color: $title-color;
+  }
+}
+
+.list-remove-style {
+  ul {
+    list-style-type: none;
+  }
+
+  li {
+    cursor: pointer;
+  }
+}
+
```

Comment:
this class does not only remove the style (how it says) but also adds a cursor pointer. why is that pointer necessary of we said that we'll use anchor tags for anything that is clickable?

Reply:
In this case, specifically, we need to have the cursor pointer over the whole row (li element) and the [routerLink]
inside which onClick opens the next component.
So when I try to wrap this element with an anchor tag or replace < li > with < a > the IDE gives me a warning 'Element a is not allowed here', so I created _list-remove-style_  class selector (maybe I should give another name as you mentioned) to be able to get appropriate styling. Also, we can use this selector in other places in a project where needed. 
But, If there is a better solution for that, I would definitely like to hear it. ;-) 

![image](https://user-images.githubusercontent.com/11729076/102892552-534c7180-4460-11eb-890e-c18c497ae074.png)

Reply:
Suggestion => Bootstrap docs: https://getbootstrap.com/docs/4.0/components/list-group/#links-and-buttons

No need for using `ul` and `li`.